README.md

```diff
@@ -17,37 +15,64 @@ engine and provide you with a set of assertions you can use to verify your proce
 
 ### Dependency
 
-Add the following dependency to your project
+#### Testcontainers (JDK 8+)
```

Comment:
:wrench: Add an introductory sentence under _Dependency_ that the dependencies depend on the Java version